dotnet/Data/WishListRepository.cs

```diff
@@ -82,34 +80,40 @@ public async Task<bool> SaveWishList(IList<ListItem> listItems, string shopperId
             };
 
             var jsonSerializedListItems = JsonConvert.SerializeObject(wishListWrapper);
-            var request = new HttpRequestMessage
+            bool IsSuccessStatusCode = false;
+            try 
             {
-                Method = HttpMethod.Patch,
-                RequestUri = new Uri($"http://{this._httpContextAccessor.HttpContext.Request.Headers[WishListConstants.VTEX_ACCOUNT_HEADER_NAME]}.vtexcommercestable.com.br/api/dataentities/{WishListConstants.DATA_ENTITY}/documents"),
-                Content = new StringContent(jsonSerializedListItems, Encoding.UTF8, WishListConstants.APPLICATION_JSON)
-            };
+                var request = new HttpRequestMessage
+                {
+                    Method = HttpMethod.Patch,
+                    RequestUri = new Uri($"http://{this._httpContextAccessor.HttpContext.Request.Headers[WishListConstants.VTEX_ACCOUNT_HEADER_NAME]}.vtexcommercestable.com.br/api/dataentities/{WishListConstants.DATA_ENTITY}/documents"),
+                    Content = new StringContent(jsonSerializedListItems, Encoding.UTF8, WishListConstants.APPLICATION_JSON)
+                };
 
-            string authToken = _context.Vtex.AuthToken;
-            if (authToken != null)
+                string authToken = _context.Vtex.AuthToken;
+                if (authToken != null)
+                {
+                    request.Headers.Add(WishListConstants.AUTHORIZATION_HEADER_NAME, authToken);
+                    request.Headers.Add(WishListConstants.VtexIdCookie, authToken);
+                    request.Headers.Add(WishListConstants.PROXY_AUTHORIZATION_HEADER_NAME, authToken);
+                }
+
+                var client = _clientFactory.CreateClient();
+                var response = await client.SendAsync(request);
+                string responseContent = await response.Content.ReadAsStringAsync();
+                IsSuccessStatusCode = response.IsSuccessStatusCode;
+                _context.Vtex.Logger.Debug("SaveWishList", null, $"'{shopperId}' '{listName}' '{documentId}' [{response.StatusCode}]\n{responseContent}");
```

Comment:
It might be good to replace this line with something like
if(!isSuccessStatusCode)
{
	_context.Vtex.Logger.Warn("SaveWishList", null, $"Failed to Save",
	new[]
	{
		("shopperId", shopperId),
		("listName", listName),
		("StatusCode", response.StatusCode),
		("responseContent", responseContent)
	});
}